src/plugins/workspace/public/components/workspace_list/index.tsx

```diff
@@ -79,46 +115,96 @@ export const WorkspaceList = () => {
       isExpander: true,
       hasActions: true,
     },
+    {
+      name: 'Actions',
+      field: '',
+      actions: [
+        {
+          name: 'Edit',
+          icon: 'pencil',
+          type: 'icon',
+          description: 'edit workspace',
+          onClick: ({ id }: WorkspaceAttribute) => handleUpdateWorkspace(id),
+        },
+      ],
+    },
   ];
 
-  const onTableChange = ({ page, sort }: CriteriaWithPagination<WorkspaceAttribute>) => {
-    const { field, direction } = sort!;
-    const { index, size } = page;
+  const workspaceCreateUrl = useMemo(() => {
+    if (!application || !http) {
+      return '';
+    }
+    return formatUrlWithWorkspaceId(
```

Comment:
How about use `cleanWorkspaceId` ?

Reply:
Could you explain more?

Reply:
Format with an empty workspace id looks a little bit strange, is that means we just want to clear workspace id here?  We can call `cleanWorkspaceId` if only want to remove workspace id.

Reply:
I don't find anything related when i search `cleanWorkspaceId` in repo.
Here I just want to generate a link to navigate to workspace create page, like [https://github.com/ruanyl/OpenSearch-Dashboards/blob/workspace/src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx#L121](url), do you have some ideas ?

Reply:
The `cleanWorkspaceId` is in `src/core/utils/workspace.ts`, maybe you need to rebase first. The `applications.getUrlForApp` was already add the basePath, call `formatUrlWithWorkspaceId` will make code more complicated.

Reply:
Ok. I think we can make this link independent of workspace id.